pkg/asset/installconfig/kubevirt/kubevirt.go

```diff
@@ -0,0 +1,130 @@
+package kubevirt
+
+import (
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/kubevirt"
+)
+
+// Platform collects kubevirt-specific configuration.
+func Platform() (*kubevirt.Platform, error) {
+	var (
+		namespace, apiVIP, ingressVIP, networkName, storageClass string
+		err                                                      error
+	)
+
+	if namespace, err = selectNamespace(); err != nil {
+		return nil, err
+	}
+
+	if apiVIP, err = selectAPIVIP(); err != nil {
+		return nil, err
+	}
+
+	if ingressVIP, err = selectIngressVIP(); err != nil {
+		return nil, err
+	}
+
+	if networkName, err = selectNetworkName(); err != nil {
+		return nil, err
+	}
+
+	if storageClass, err = selectStorageClass(); err != nil {
+		return nil, err
+	}
+
+	return &kubevirt.Platform{
+		Namespace:    namespace,
+		StorageClass: storageClass,
+		APIVIP:       apiVIP,
+		IngressVIP:   ingressVIP,
+		NetworkName:  networkName,
+	}, nil
+}
+
+func selectNamespace() (string, error) {
+	var selectedNamespace string
+
+	err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Namespace",
+				Help:    "The namespace, in the undercluster, where all the resources of the overcluster would be created.",
+			},
+		},
+	}, &selectedNamespace)
+
+	return selectedNamespace, err
+}
+
+func selectAPIVIP() (string, error) {
+	var selectedAPIVIP string
+
+	defaultValue := ""
+
+	err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "API VIP",
+				Help:    "An IP which will be served by bootstrap and then pivoted masters, using keepalived.",
```

Comment:
this help message should be similar to ingressVIP. mentioning bootstrap and pivot is internal info

Reply:
Fixed to:
`IP for the internal kubernetes api, that all components will interact with`